README.md

```diff
@@ -6,76 +6,86 @@
+**IMPORTANT:** As of version 1.7 Dashboard uses minimal privileges setup by default. It means, that some manual steps
+are required to make it fully secure and functional. Before performing any steps read [Access Control](
+https://github.com/kubernetes/dashboard/wiki/Access-control) guide.
+
+### Recommended setup
+
+Full security can be ensured only by accessing Dashboard over HTTPS. In order to enable HTTPS mode certificates need
+to be passed to the application. This can be achieved by providing `--tls-cert-file` and `--tls-cert-key` flags to the
+Dashboard. 
+
+Assuming that you have `dashboard.crt` and `dashboard.key` files stored under `$HOME/certs` directory,
+you need to create config map with contents of these files:
+
+```bash
+kubectl apply configmap kubernetes-dasboard-certs --from-file=$HOME/certs -n kube-system
 ```
 
-If you are using Dashboard on a cluster with disabled authorization, you can install the latest stable release by running the following command:
-```shell
-$ kubectl create -f https://git.io/kube-dashboard-no-rbac
+Afterwards, you are ready to deploy Dashboard using following command:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/master/src/deploy/recommended/kubernetes-dashboard.yaml
 ```
 
-You can also install unstable HEAD builds with the newest features that the team works on by
-following the [development guide](docs/devel/head-releases.md).
+### Alternative setup
 
-Note that for the metrics and graphs to be available you need to
-have [Heapster](https://github.com/kubernetes/heapster/) running in your cluster.
+This setup is not fully secure, however it does not require any additional steps. In this setup access control can be
+ensured only by using [Authorization Header](
+https://github.com/kubernetes/dashboard/wiki/Access-control#authorization-header) feature. To deploy Dashboard execute
+following command:
 
-## Usage
-The easiest way to access Dashboard is to use kubectl. Run the following command in your desktop environment:
-```shell
-$ kubectl proxy
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/master/src/deploy/alternative/kubernetes-dashboard.yaml
 ```
-kubectl will handle authentication with apiserver and make Dashboard available at [http://localhost:8001/ui](http://localhost:8001/ui)
 
-The UI can _only_ be accessed from the machine where the command is executed. See `kubectl proxy --help` for more options.
+**NOTE:** You can also install latest development builds with the newest features that the team works on by
+following the [Development Release](https://github.com/kubernetes/dashboard/wiki/Installation#development-release)
+guide.
 
-## Alternative Usage
-You may access the UI directly via the apiserver proxy. Open a browser and navigate to `https://<kubernetes-master>/ui`.
+**NOTE:** You need to have [Heapster](https://github.com/kubernetes/heapster/) running in your cluster for the metrics
```

Comment:
One thing to these notes. When I actually look at the whole page then they seem to be related to 'Alternative Setup' section only. It does not look like they are related to the whole 'Deloyment' section.